modules/vaos/app/schemas/appointments_index.json

```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "required": [
+    "data"
+  ],
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
```

Comment:
The documentation did not actually specify anything as required, but I had a list of required values in a previous spike. I had to remove some from the spike due to spec failures but these worked for all specs. If anyone has reason to think these should not be required, let me know and I'll remove them.